skimage/exposure/hdr.py

```diff
@@ -0,0 +1,351 @@
+import numpy as np
+import warnings
+
+
+def make_hdr(images, exposure, radiance_map, depth=16, channel_axis=None):
+    """
+    Compute the HDR image from a series of images with a given radiance
+    mapping.
+
+    Parameters
+    ----------
+    images: list of numpy arrays, or numpy ndarray or ImageCollection
+        List of images in the form of numpy arrays, or a numpy array with the
+        first dimension being the different images.
+        Either greyscale or colour (RGB). Can't be float images
+        List of images as numpy arrays. Either greyscale or colour
+        (RGB).
+        Can also be an ImageCollection
+    exposure : numpy 1D array
+        Array of exposure times in seconds.
+        Images some times have these in the exif information.
+    radiance_map : numpy array
+        Array (idx) mapping counts to radiance value, if input is RGB this must
```

Comment:
Map which counts to radiance values?

Reply:
The camera counts, as per what a CCD/CMOS actually measures (normally number of electrons)